docs/asciidoc/coreFeatures.adoc

```diff
@@ -47,7 +47,11 @@ The following image shows how a `Mono` transforms an item:
 image::https://raw.githubusercontent.com/reactor/reactor-core/v3.0.7.RELEASE/src/docs/marble/mono.png[Mono]
 
 A `Mono<T>` is a specialized `Publisher<T>` that emits at most one item and then
-(optionally) terminates with an `onComplete` signal or an `onError` signal.
+terminates with an `onComplete` signal or alternatively, emits an `onError` signal.
```

Comment:
It might be still a bit ambiguous for someone less familiar with Reactive Streams (this could be their first exposure to the concepts).

Suggestion:
```
that emits at most one item via the `onNext` signal then terminates with an `onComplete` signal (successful `Mono`, with or without value), or only emits a single `onError` signal (failed `Mono`).
```